cmd/copy.go

```diff
@@ -1250,7 +1259,7 @@ func init() {
 	rootCmd.AddCommand(cpCmd)
 
 	// filters change which files get transferred
-	cpCmd.PersistentFlags().BoolVar(&raw.copyContents, "strip-top-dir", false, "copy everything under the source, not the source itself")
+	cpCmd.PersistentFlags().BoolVar(&raw.stripTopDir, "strip-top-dir", false, "strip the source's root folder from the destination path, akin to \"cp dir/*\" (ex. sourcedir/subdir1/file1 copies to subdir1/file1 on destination)")
```

Comment:
Maybe add "whereas without strip-to-dir, it would copy to sourcedir/subdir1/file1 on the destination"